### PR TITLE
categorize create external account ops as Sent

### PR DESCRIPTION
**Pull request by piyalbasu**
https://user-images.githubusercontent.com/6789586/183221527-541a9308-8d4a-4878-b23a-26e96f16a6e9.mov

The first op (`Create Account`) is the account being created.
The second `XLM` is me sending 5 XLM to another account to create it

### CODE REVIEW

File: @shared/api/types.ts [hunk 1]
```diff
@@ -71,12 +71,7 @@ export interface AssetIcons {
 export type Balances = Types.BalanceMap | null;
 
 /* eslint-disable camelcase */
-export type HorizonOperation = (
-  | Horizon.PaymentOperationResponse
-  | Horizon.PathPaymentOperationResponse
-) & {
-  transaction_attr: Horizon.TransactionResponse;
-};
+export type HorizonOperation = any;
```
Comment (piyalbasu) [inline @shared/api/types.ts:74]: @acharb Ugh, I'm realizing I did a pretty bad job here. This doesn't take into account other Response types (like `CreateAccountOperationResponse`). When I add other types, TS doesn't do a great job of inferring the correct type when we use our helper methods like `getIsPayment`. I need to rethink this 🤔 

Associated ticket:
https://stellarorg.atlassian.net/browse/WAL-409

**Comment by piyalbasu** [comment]
https://user-images.githubusercontent.com/6789586/183223234-0e303e65-6dc6-49d4-8c75-34eead13e22f.mov

**Review by acharb — APPROVED** [review]
lgtm!

File: @shared/api/types.ts (hunk 1 above)
Comment (acharb) [reply @shared/api/types.ts:74]: I think this is fine for now, agreed we should probably take another look at this, along with the Balances types

**Pull request merged**